client/components/site-selector/style.scss

```diff
@@ -160,6 +160,10 @@
 	padding: 10px 20px;
 }
 
+.layout__secondary .site-selector__no-results {
```

Comment:
this should probably go in `components/layout/style.scss` since it's an override for that component, not a core style of the site selector.

Reply:
Moved all the `.layout__secondary` overrides to `client/layout/style.scss` for consistency!

Reply:
FWIW, #31642 addresses this somewhat too.